README.md

```diff
@@ -40,16 +40,93 @@ The Rucio file catalog is much flatter than that of a usual `POSIX` filesystems
 - Fix Mac OS X mouting issues
 - Surely a lot more...
 
-## How to build
+## Getting Started
+
+### Cloning the repository
+Fork the repository and clone it on your local machine for development
+```BASH
+$ git clone https://github.com/<your-username>/fuse-posix.git
+$ cd fuse-posix/
+$ git remote add upstream https://github.com/rucio/fuse-posix.git
+```
+### Install the dependencies
+Next step is to install the dependencies which shall be required to build the software
+
+**Ubuntu/Debian (apt)**
+```BASH
+$ sudo apt-get install cmake git g++ libfuse-dev libcurl4-gnutls-dev
+```
+**CentOS (yum)**
+```BASH
+$ yum install -y cmake3 libcurl-devel libfuse-devel
+```
+
+### Build the software
 To build the software please run:
 
 ```[shell]
-./build.sh
+$ cd fuse-posix/
+$ ./build.sh
+```
+**NOTE:** To complete the build `libcurl-devel`, `fuse-libs` and `fuse-devel` packages (or equivalent) must be present:
+`cmake` will try to locate them for you and trigger some build messages if unable to do so.
+Please note that `cmake` version 3 or greater is needed.
+
+### Setting up the FUSE mount
+If you're Ubuntu 18.04 LTS or above, you should particularly follow this step to ensure the right working of the Rucio-FUSE mount.
```

Comment:
I would avoid this kind of specificity. This steps will be needed even on newer versions of existing Linux flavours. Just purge line 76!

Reply:
Okay got it! Done!